rest/db-file-get.rst

```diff
@@ -1,10 +1,27 @@
 GET /rest/db/file
 =================
 
-Returns most data available about a given file, including version and
-availability. Takes ``folder`` and ``file`` parameters. ``local`` and
-``global`` refer to the current file on disk and the globally newest file,
-respectively.
+Returns data about a given file, including version and
+availability.
+
+Request
+-------
+
+The call requires parameters for `folder` and `file` in the querystring:
+
+- `folder` is the folder code which you can find in the synthing UI, e.g. `5camp-slpa8`
```

Comment:
It's called ID in the UI and other docs, so I'd use that here too.

```suggestion
- `folder` is the folder ID which you can find in the synthing UI, e.g. `5camp-slpa8`
```

Reply:
applied, thanks!